src/managers/CacheManager.js

```diff
@@ -0,0 +1,52 @@
+class CacheManager extends Map {
+    /**
+     * Creates a new CacheManager instance.
+     * @param {Number} ttl The time to live for the cache in minutes.
```

Comment:
Might be worth to clean this up a bit - ttl is not a parameter of the constructor.

Reply:
Remplaced by the correct value 😅